services/api-server/openapi.json

```diff
@@ -724,14 +724,14 @@
         ]
```

Comment:
@bisgaard-itis is it possible that somebody before forgot to run make openapi-specs? i don't know why this file changed.

Reply:
strange ... there should be a test that checks openapi.json vs app.schema() somewhere to prevent this.

Reply:
This is really bad. We had this test at some point. All fastapi service have one. @bisgaard-itis do you remember refactoring it?
Can we please add it again. We can reuse this from another service https://github.com/ITISFoundation/osparc-simcore/blob/f2c5bb0bc240f6e11ca6732bf4bd4f841032f74a/services/payments/tests/unit/api/test__oas_spec.py

@bisgaard-itis how do we know now that the client is correct?

Reply:
Hmm, this is strange. I don't remember refactoring any tests. But I can look into it. We have the e2e client tests which test compatibility. Of course they don't test everything, but they test a lot of the new functionality on master. So I think the client should be ok. The change looks like a reshuffeling to me. Could it be some dependency which changed? Maybe because generating this `openapi.json` uses a docker image using the `latest` tag? Ah or it could be because I did a refactoring in my last PR. One of the bots complained that the python file with solver jobs was too long, so I had to refactor it into two files. That probably reshuffled the order of the methods which is why it appears shuffled here. I think that is likely the reason. But I think this is a good occasion to make sure our testing of this is done properly.

Reply:
Ye that might be the case! ok so just let me know whether i have green to merge it to master :) thanks